exercises/split-second-stopwatch/instructions.md

```diff
@@ -0,0 +1,22 @@
+# Instructions
+
+Your task is to build a stopwatch to keep precise track of lap times.
+
+The stopwatch uses four commands (Start, Stop, Lap, and Reset) to keep track of:
```

Comment:
The commands here are capitalized. Should they be?

Reply:
No. Updated.